doc/select.md

```diff
@@ -0,0 +1,84 @@
+# Select examples
+
+## Pagination
+
+Select supports pagination. To use it, use the [``after``](https://github.com/tarantool/crud#select) and [``first``](https://github.com/tarantool/crud#select) parameter.
```

Comment:
There is no such section for pairs. It would be great to list all possible options and then show how they help for users.